CHANGELOG.md

```diff
@@ -2,7 +2,7 @@ x.x.x Release notes (yyyy-MM-dd)
 =============================================================
 
 ### Breaking change
-* Model classes passed as schema to the `Realm` constructor must now extend `Realm.Object`.
+* Model classes passed as schema to the `Realm` constructor must now extend `Realm.Object` and will no longer have their constructors called when pulling an object of that type from the database.
```

Comment:
Do you think there'd be any value in adding a small example of how to migrate a class to the new style?

Reply:
I've added an example of usage to the changelog.